src/history/import_csv_to_db.py

```diff
@@ -16,7 +16,7 @@ def process_csv_files(csv_files: list[str], is_cleaning: bool = False) -> None:
 
     for csv_file in csv_files:
         # Get the name of the table from the filename
-        table_name = (Path(csv_file).parent / Path(csv_file).root).replace("/", "_")
+        table_name = Path(str(Path(csv_file).parent / Path(csv_file).root).replace("/", "_"))
```

Comment:
The `table_name` is a `str`. Is there a need to wrap it in a `Path()` constructor outside again?